tcms/testplans/tests/tests.py

```diff
@@ -148,6 +148,12 @@ def test_add_cases_sortkey_autoincrement(self):
         case_plan = plan.add_case(TestCaseFactory(), sortkey=15)
         self.assertEqual(15, case_plan.sortkey)
 
+    def test_get_full_url(self):
+        test_plan_url = self.plan_1.get_full_url()
+        response = self.client.get(test_plan_url)
+        self.assertIsNotNone(response, "Empty response")
```

Comment:
If you want to pass value for the `msg` parameter then it needs to be named, like shown in the signature so it is clear what that is:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertIsNotNone

However a second parameter isn't really necessary b/c the assertion is clear enough by itself.

Reply:
Response can't be None anyway so this assertion isn't very helpful.